addons/mrp/report/mrp_report_bom_structure.py

```diff
@@ -103,6 +103,8 @@ def _get_bom(self, bom_id=False, product_id=False, line_qty=False, line_id=False
             'level': level or 0,
             'operations': operations,
             'operations_cost': sum([op['total'] for op in operations]),
+            'attachments': self.env['mrp.document'].search(['|', '&',
+                ('res_model', '=', 'product.product'), ('res_id', '=', product.id), '&', ('res_model', '=', 'product.template'), ('res_id', '=', product.product_tmpl_id.id)]),
```

Comment:
@sle-odoo Is it worth to create an helper method that take the product.product as argument and return all the linked mrp.documents. Just to avoid this duplicated code?